internal/app/handlers.go

```diff
@@ -470,3 +523,26 @@ func (app *Application) handleBannerToggle(w http.ResponseWriter, r *http.Reques
 	component := components.UsaBanner(action)
 	component.Render(r.Context(), w)
 }
+
+func handleError(err error, input string, logger *slog.Logger, w http.ResponseWriter, r *http.Request) {
```

Comment:
Should/can this function be moved to the errors package?

Reply:
done!